### ISSUE TITLE
No Python 3 supported

### ISSUE THREAD

**Issue by deeplook** [raddevon/flask-permissions#11]
I'm getting the following error on Python 3 (while the `setup.py` file makes no claims about supported Python versions):

```
  File "/Users/dinu/miniconda3/lib/python3.5/site-packages/flask_permissions/models.py", line 112, in __init__
    if roles and isinstance(roles, basestring):
NameError: name 'basestring' is not defined
```

Is this still being developed?

**Comment by raddevon** [comment]
I don't work much in Python these days, and I have not worked on flask-permissions in a long while. It does not have support for Python 3, but I'd gladly take a look at a pull request if you want to submit one.

**Comment by deeplook** [comment]
Well, I'm using this workaround in `models.py`:

```
        if roles:
            import sys
            v = sys.version_info.major
            if v == 2 and isinstance(roles, basestring):
                roles = [roles]
            elif v == 3 and isinstance(roles, str):
                roles = [roles]
```

**Comment by ElFitaFlores** [comment]
Please close this issue, fixed in commit 3481373.
